src/egress_assess.yml

```
@@ -0,0 +1,14 @@
+---
+- hosts: all
+  name: Install Egress-Assess tool
```

I'm fine with the playbook name remaining the same but I think this descriptor should reflect the tool being installed.

```suggestion
  name: Install Auto-Egress-Assess tool
```